src/components/Calculator.css

```diff
@@ -1,4 +1,5 @@
 .calculator {
+  margin-top: 25vh;
   width: 450px;
   box-shadow: 0 0 10px rgba(0, 0, 0, 0.3);
   display: inline-block;
```

Comment:
- You have done great so far :+1: :+1: but there are a few changes you need to make before moving to the next milestone. :+1:
> The general layout should match these wireframes: 
- [ ] As stated in the project requirement, your design must match the given wireframe, looking at the wireframe, you can see that the calculator UI, is aligned to the right with the text `Let's do some maths` aligned by the left, this is not the case in your own design. I'll suggest you update yours. :smile: :small_airplane: 

- - WIREFRAME:
![Selection_009](https://github.com/SaveryIV/Math-Magicians/assets/107632780/f3fafc18-3b53-4df5-afaf-607435cbeed6)


- - YOUR DESIGN:
![Screenshot from 2023-08-23 10-10-50](https://github.com/SaveryIV/Math-Magicians/assets/107632780/8104a9aa-ea48-4da6-a3e7-54712b7d335c)